### PR TITLE
[4.x] Add README

### PR DESCRIPTION
**Pull request by tjrgg**
### Items Changed

- [ ] Commands
- [ ] Events
- [ ] Handlers
- [ ] Structures
- [ ] Tasks
- [x] Documentation
- [ ] Other: ______

### Description

- Add `all-contributors-cli` as a development dependency to manage contributors list
- Add README

### Issues

N/A

### Have You...?

- [x] I have read the [Contributing Guidelines](https://github.com/typicalbot/typicalbot/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [pull request template](https://github.com/typicalbot/typicalbot/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
- [x] I have checked open [pull requests](https://github.com/typicalbot/typicalbot/pulls) for upcoming features and fixes.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "projectName": "typicalbot",
+  "projectOwner": "sylkellc",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 50,
+  "commit": false,
+  "commitConvention": "angular",
```
Comment (nsylke) [inline .all-contributorsrc:11]: This should be standard or conventional, as conventional commit doesn’t follow angular spec entirely.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<p align="center">
+    <h1 align="center">TypicalBot</h1>
+</p>
+<p align="center">
+    <i>A stable, lightning fast, easy to use, <a href="https://discordapp.com">Discord</a> bot built using <a href="https://discord.js.org">Discord.js</a> and <a href="https://nodejs.org">Node.js</a>.<br>Try out TypicalBot using our hosted version at <a href="https://typicalbot.com/invite">typicalbot.com/invite</a>.</i>
+</p>
+<p align="center">
+    <a href="https://discord.gg/typicalbot"><img src="https://discordapp.com/api/guilds/163038706117115906/embed.png?style=shield" alt="Discord"></a>
```
Comment (nsylke) [inline README.md:10]: Let’s add the GH action badges.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+- [Introduction](#introduction)
+- [Table of Contents](#table-of-contents)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Contributors](#contributors)
+- [Show Your Support](#show-your-support)
+- [License](#license)
+## Getting Started
```
Comment (nsylke) [inline README.md:28]: Add new line before this heading

File: .all-contributorsrc (hunk 1 above)
Comment (tjrgg) [reply .all-contributorsrc:11]: I was given neither as an option. Regardless, as I understand it, this setting has no effect as I'm the app isn't set to auto-commit.

**Pull request closed without merging**